src/table/components/TableHeadWrapper.jsx

```diff
@@ -18,16 +19,21 @@ function TableHeadWrapper({
   translator,
   selectionsAPI,
   keyboard,
-  children,
 }) {
   const { columns, paginationNeeded } = tableData;
+  const setHeadRowHeight = useContextSelector(TableContext, (value) => value.setHeadRowHeight);
   const isFocusInHead = useContextSelector(TableContext, (value) => value.focusedCellCoord[0] === 0);
   const setFocusedCellCoord = useContextSelector(TableContext, (value) => value.setFocusedCellCoord);
   const headerStyle = useMemo(() => getHeaderStyle(layout, theme), [layout, theme]);
+  const headRowRef = useRef();
+
+  useEffect(() => {
+    setHeadRowHeight(headRowRef.current.clientHeight);
+  }, [headRowRef.current, headerStyle.fontSize]);
```

Comment:
```suggestion
  useEffect(() => {
    setHeadRowHeight(headRowRef.current.getBoundingClientRect().height);
  }, [headRowRef.current, headerStyle.fontSize]);
```

Reply:
reference: https://developer.mozilla.org/en-US/docs/Web/API/Element/getBoundingClientRect

Reply:
Yes! good idea! It's more accurate ! I'll replace it with clientHeight! However the issue with the tiny gap between rows comes from somewhere else I think. There should be something related to the first and last child styling because this misbehavior can only be seen on the last header cell. I actually saw it on the first cell as well.